app/src/main/java/com/cornellappdev/android/eatery/ui/screens/HomeScreen.kt

```diff
@@ -227,32 +227,31 @@ fun HomeScreen(
                             onSearchClick = onSearchClick,
                             onEateryClick = onEateryClick,
                             onFavoriteExpand = onFavoriteExpand,
-                            onNearestExpand = onNearestExpand,
                             modalBottomSheetState = modalBottomSheetState,
-                            eateriesApiResponse = eateriesApiResponse,
-                            favorites = favorites,
-                            nearestEateries = nearestEateries,
-                            filters = filters,
-                            onFavoriteClick = { eatery, favorite ->
-                                if (favorite) {
-                                    homeViewModel.addFavorite(eatery.id)
-                                } else {
-                                    homeViewModel.removeFavorite(eatery.id)
-                                }
-                            },
-                            onFilterClicked = { filter ->
-                                if (filters.contains(filter)) {
-                                    homeViewModel.removeFilter(filter)
-                                } else {
-                                    homeViewModel.addFilter(filter)
-                                }
-                            },
-                            onResetFilters = {
-                                homeViewModel.resetFilters()
-                            }
-                        )
+                    eateriesApiResponse = eateriesApiResponse,
+                    favorites = favorites,
+                    nearestEateries = nearestEateries,
+                    filters = filters,
+                    onFavoriteClick = { eatery, favorite ->
+                        if (favorite) {
+                            homeViewModel.addFavorite(eatery.id)
+                        } else {
+                            homeViewModel.removeFavorite(eatery.id)
+                        }
+                    },
+                    onFilterClicked = { filter ->
+                        if (filters.contains(filter)) {
+                            homeViewModel.removeFilter(filter)
+                        } else {
+                            homeViewModel.addFilter(filter)
+                        }
+                    },
+                    onResetFilters = {
+                        homeViewModel.resetFilters()
```

Comment:
Looks like the indentation here became a little sus. Could you reformat file? Should be command option L on mac (or something like that...)

Reply:
Done!